var/spack/repos/builtin/packages/py-antspyx/package.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyAntspyx(PythonPackage):
+    """Advanced Normalization Tools in Python."""
+
+    homepage = "https://pypi.org/project/antspyx/"
+    url = "https://github.com/ANTsX/ANTsPy/archive/refs/tags/v0.2.7.tar.gz"
```

Comment:
https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#pypi-vs-github

Reply:
Sorry, just rebased in a brain-dead fashion.